tests-integration/tests/core/ics02_client/update_client.rs

```diff
@@ -1431,7 +1431,7 @@ fn test_expired_client() {
         );
 
     while ctx.ibc_store.host_timestamp().expect("no error")
-        < (timestamp + trusting_period).expect("no error")
+        <= (timestamp + trusting_period).expect("no error")
```

Comment:
Why is this required? I think, the unchanged version is consistent with this PR: _until the latest header is valid, produce blocks_.

Reply:
Ahh, correct. [a32ad1c](https://github.com/cosmos/ibc-rs/pull/1287/commits/a32ad1cbe2f5cd3ae08418db7a7e26df4071d1a1)